src/randoop/main/TypeReader.java

```diff
@@ -0,0 +1,102 @@
+package randoop.main;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+import randoop.types.TypeNames;
+import randoop.util.Files;
+
+import plume.EntryReader;
+
+/**
+ * ClassReader is a library of static methods that converts a list of
```

Comment:
Class is named `TypeReader`.  However, since it returns a list of `Class` objects, renaming it to `ClassReader` makes sense to me.